jenkins-admin/managed-config-files/custom/jjbini/content

```diff
@@ -0,0 +1,21 @@
+[job_builder]
+ignore_cache=True
+keep_descriptions=False
+include_path=.:scripts:~/git/
+recursive=True
+
+[jenkins]
+url=https://jenkins-new.magmacore.org/
```

Comment:
I think tokens should be all caps right?
In this case, your token should be "LFJOBBUILDER"